src/index.js

```diff
@@ -190,6 +153,53 @@ class Pathfinding {
 	}
 }
 
+/**
+ * Returns closest node group ID for given position.
+ * @param  {string} zoneID
+ * @param  {THREE.Vector3} position
+ * @return {number}
+ */
+Pathfinding.prototype.getGroup = (function() {
+	const plane = new THREE.Plane();
+	return function (zoneID, position, checkPolygon = false) {
+		if (!this.zones[zoneID]) return null;
+
+		let closestNodeGroup = null;
+		let distance = Math.pow(50, 2);
+		const zone = this.zones[zoneID];
+
+		for (let i = 0; i < zone.groups.length; i++) {
+			const group = zone.groups[i];
+			for (const node of group) {
+				if (checkPolygon) {
+					plane.setFromCoplanarPoints(
+						zone.vertices[node.vertexIds[0]],
+						zone.vertices[node.vertexIds[1]],
+						zone.vertices[node.vertexIds[2]]
+					);
+					if (Math.abs(plane.distanceToPoint(position)) < 0.0001) {
```

Comment:
I don't think it needs to be this strict, maybe just `0.01`? One centimeter, taking 1 unit = 1 meter here.